server/db.json

```
@@ -0,0 +1,204 @@
+{
+  "waste-items": [
+    {
+      "id": 1301,
+      "description": "dropped icecream",
+      "type": "residual",
+      "imageSrc": "/residual_01.svg",
```

You converted your images in `.png`. 
```suggestion
      "imageSrc": "/residual_01.png",
```